src/test/java/web/tests/FlipkartTest.java

```diff
@@ -27,7 +29,8 @@ public void testSortFilter(String product, String pagelimit) {
             if (page != 1) {
                 searchPage.selectPageNumber(page);
             }
-            Assertions.assertTrue(searchPage.countingPagePrices(), "prices not matching");
+           softAssert.assertThat(searchPage.countingPagePrices()).isTrue();
```

Comment:
add description if assert fails using this syntax : 
softAssert.assertThat(something).as("description").isTrue();